crates/typst/src/math/attach.rs

```diff
@@ -469,3 +515,33 @@ fn compute_shifts_up_and_down(
 fn is_integral_char(c: char) -> bool {
     ('∫'..='∳').contains(&c) || ('⨋'..='⨜').contains(&c)
 }
+
+fn stretch_fragment(
+    ctx: &mut MathContext,
+    styles: StyleChain,
+    stretch: StretchInfo,
+    relative_to_width: Abs,
+    fragment: MathFragment,
+) -> MathFragment {
```

Comment:
I guess if we're passing in the fragment already, we may as well pass it with a mutable reference and instead not return anything.